RELEASES.md

```diff
@@ -2,19 +2,28 @@
 
 After review of a protocol-branch and before each release the following steps are necessary:
 
+1.  with the protocol-branch checked-out:
 
-1. with the protocol-branch checked-out:
-    1. bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`. Some examples:
-        - a new protocol added in 2020 that was released in 2020 (fifth release in that year): `2020.00.dev` to `2020.05`
-        - an update of the `2020.05` protocol in 2021 (first release in that year): `2020.05.dev` to `2021.01`
-        - an update of the `2020.02` version of a protocol in the same year (fourth release in that year): `2020.02.dev` to `2020.04`
-    1. update the `.zenodo.json` file:
-        - add new authorships
-    1. update the repo `NEWS.md` file
-1. merge the branch to the master and add general and specific tags (see [release model](README.md#release-model)):
-    1. general tag: `protocols-YYYY.NN`
-    1. specific tag: `protocol-code-YYYY.NN`
-1. Check if continuous integration proceeded without problems
-1. Make a GitHub release from the general tag
-1. Check success of publication steps at protocols.inbo.be and at Zenodo (records both source and rendered) 
+    1.  bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`. Some examples:
+
+        -   a new protocol added in 2020 that was released in 2020 (fifth release in that year): `2020.00.dev` to `2020.05`
+        -   an update of the `2020.05` protocol in 2021 (first release in that year): `2020.05.dev` to `2021.01`
+        -   an update of the `2020.02` version of a protocol in the same year (fourth release in that year): `2020.02.dev` to `2020.04`
+
+    2.  update the `.zenodo.json` file:
+
+        -   add new authorships
+
+    3.  update the repo `NEWS.md` file
+
+2.  merge the branch to the master and add general and specific tags (see [release model](README.md#release-model)):
+
+    1.  general tag: `protocols-YYYY.NN`
+    2.  specific tag: `protocol-code-YYYY.NN`
+
+3.  Check if continuous integration proceeded without problems
+
+4.  Make a GitHub release from the general tag
+
+5.  Check success of publication steps at protocols.inbo.be and at Zenodo (records both source and rendered)
```

Comment:
It's noteworthy that between all items a whiteline has been added, and that they have been renumbered. At least when rendered at GitHub, this results in whitespace between lines. Is this due to VME or to the markdown RProj options?

Reply:
Yes, this happened in 4d596e4 after setting R project options to wrap: sentence and canonical: true for the markdown editor options in ca3c164. The added whitespace and renumbering is likely due to the "canonical" setting.